src/test/java/UnitTest.java

```diff
@@ -0,0 +1,179 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class UnitTest {
+    private StringEvaluation stringEvaluation;
+    private String testInput = null;
+    private ByteArrayOutputStream outputStreamCaptor;
+    private InputStream in;
+
+    @BeforeEach
+    protected void setUp(){
```

Comment:
Some tests are failing when I run these locally, more than likely due to whitespace issues https://www.baeldung.com/java-compare-string-whitespace.
![image](https://user-images.githubusercontent.com/90250484/191703377-f230d25e-cb05-4bf4-818a-9be48a3552c1.png)